### ISSUE TITLE
Investigate why python-pip3 has to be sometimes reinstalled

### ISSUE THREAD

**Issue by ekristen** [teamdfir/sift#266]
Some has happened with python that causes it to not work properly. Python 2 pip has to be below 10, but this seems to cause problems with python3-pip as well.

**Comment by AtesComp** [comment]
In my instance, I downloaded the latest SIFT Workstation VM.
```
sudo apt upgrade
sudo apt update
```
I got upwards of 50+ errors trying to do a SIFT upgrade after this.  All pips must be at version 9.0.1 to successfully upgrade SIFT.  Doing an `apt upgrade` may upgrade pip2 and pip3 to 10.0.1.  Of course, various `pip install --upgrade pip` will mess it up as well (I "might" have done this as well ;).

To downgrade, this works for pip2:
```
sudo pip2 install pip==9.0.1
```
However, pip3 fails using sudo.  Using the --user option (as suggested) works:
```
pip3 install pip==9.0.1 --user
```
Then,
```
sudo sift upgrade
```
produced 0 errors.

The `sift version' command now produces:
```
> sift-cli@1.6.1-master.bde3e23
> sift-version: v2018.16.0
```
